<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWidget</class>
 <widget class="QWidget" name="CalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>626</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioAutomatic">
        <property name="text">
         <string>Automatic calibration</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioImport">
        <property name="text">
         <string>Import a template</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutModes">
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>302</width>
        <height>554</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageAutomatic">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QStackedWidget" name="automaticCalibStackedWidget">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="pageParameters">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QGroupBox" name="boxRig">
                 <property name="title">
                  <string>Rig selection</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_10">
                  <item>
                   <widget class="RigWidget" name="rigWidget" native="true"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deshuffleButton">
                 <property name="text">
                  <string>Deshuffle legacy algorithm (beta)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="boxFrames">
                 <property name="title">
                  <string>Frame selection</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <item>
                   <widget class="QRadioButton" name="radioFrameAutomatic">
                    <property name="text">
                     <string>Automatic frame selection on the sequence</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioFrameManual">
                    <property name="text">
                     <string>Manual frame selection</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="TimeSequenceWidget" name="timeSequence" native="true"/>
                  </item>
                  <item>
                   <widget class="QWidget" name="manualInputFramesWidget" native="true">
                    <layout class="QGridLayout" name="gridLayout">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item row="1" column="0" rowspan="4">
                      <widget class="QListWidget" name="listFrames">
                       <property name="frameShape">
                        <enum>QFrame::NoFrame</enum>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QPushButton" name="addFrameButton">
                       <property name="text">
                        <string>Add current frame</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QPushButton" name="removeFrameButton">
                       <property name="text">
                        <string>Remove selected</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="1">
                      <widget class="QPushButton" name="clearFramesButton">
                       <property name="text">
                        <string>Clear all</string>
                       </property>
                      </widget>
                     </item>
                     <item row="4" column="1">
                      <spacer name="verticalSpacer">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>0</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="cropButton">
                 <property name="text">
                  <string>Adjust inputs crop</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deshuffleOnlyOnListButton">
                 <property name="text">
                  <string>Deshuffle only (beta)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="syntheticKeypointsCheckBox">
                 <property name="text">
                  <string>Generate synthetic keypoints (beta)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="calibrateButton">
                 <property name="text">
                  <string>Calibrate</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="pageResults">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelUsedRig">
                 <property name="text">
                  <string notr="true">TextLabel for used rig</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="framesSelected">
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QCheckBox" name="checkDisplayFrames">
                    <property name="text">
                     <string>Display frames</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QListWidget" name="listUsedFrames">
                    <property name="frameShape">
                     <enum>QFrame::NoFrame</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="cropButton_inImprove">
                 <property name="text">
                  <string>Adjust inputs crop</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="syntheticKeypointsCheckBox_InImprove">
                 <property name="text">
                  <string>Generate synthetic keypoints (beta)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="improveOnCurrentFrameButton">
                 <property name="text">
                  <string>Adjust using current frame</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deshuffleOnlyOnCurrentFrameButton">
                 <property name="text">
                  <string>Deshuffle only (beta)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="resetButton">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageImport">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QStackedWidget" name="stackedImport">
             <widget class="QWidget" name="pageSelection">
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="applyCalibrationButton">
                 <property name="toolTip">
                  <string>Search and apply a calibration template from PTGui, Hugin, or a VideoStitch project</string>
                 </property>
                 <property name="text">
                  <string>Browse calibration...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="recentCalibrationsButton">
                 <property name="toolTip">
                  <string>Select and apply a recent calibration template</string>
                 </property>
                 <property name="text">
                  <string>Recent calibrations</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="pageImportResults">
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="labelRigName">
                 <property name="text">
                  <string>Rig used for calibration: Imported</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="buttonResetTemplate">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimeSequenceWidget</class>
   <extends>QWidget</extends>
   <header>../src/widgets/timesequence.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RigWidget</class>
   <extends>QWidget</extends>
   <header>libvideostitch-gui/widgets/rigwidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
